plugin.rb

```diff
@@ -23,7 +23,7 @@
   if GlobalSetting.try(:saml_slo_target_url).present?
     SiteSetting.class_eval do
       def self.logout_redirect
-        Discourse.base_url + "/auth/saml/spslo"
+        SamlAuthenticator.saml_base_url + "/auth/saml/spslo"
```

Comment:
This is weird to me. `saml_base_url` is not a class method so how does this work?